### ISSUE TITLE
Insert the logo in the navbar

### ISSUE THREAD

**Issue by Raghwendra-Dey** [Raghwendra-Dey/productivity_meter#80]
Insert the logo before the `productivity_meter` text in the navbar. Try out both of the logos in the assets folder of the main repo:master, and post screenshots of how it looks, then we would decide later, which one to pick.

**Comment by Raghwendra-Dey** [comment]
@Nishant2907 would you like to take this up?

**Comment by Nishant2907** [comment]
@Raghwendra-Dey yes sure, i can do it.

**Comment by Raghwendra-Dey** [comment]
@Nishant2907 please go ahead!

**Comment by Raghwendra-Dey** [comment]
Hey @Nishant2907 are you done with adding the logo?

**Comment by Nishant2907** [comment]
Yes i have done, just forgot to make a pr. Sorry for that. Will make a Pr today itself

**Comment by Raghwendra-Dey** [comment]
Okay! Cool!

**Comment by Nishant2907** [comment]
Hey @Raghwendra-Dey , the logo is added, but i don't think that the available logo might look good with the background
![image](https://user-images.githubusercontent.com/59792198/103421135-df793a00-4bc0-11eb-8e3d-60fe7d18d301.png)

**Comment by Raghwendra-Dey** [comment]
Maybe try with the other one?

**Comment by Nishant2907** [comment]
Have a look at this one. It also doesn't seem much good.
![image](https://user-images.githubusercontent.com/59792198/103421370-f5d3c580-4bc1-11eb-9d37-01aa3dbc16c7.png)

**Comment by Raghwendra-Dey** [comment]
> Have a look at this one. It also doesn't seem much good.
> ![image](https://user-images.githubusercontent.com/59792198/103421370-f5d3c580-4bc1-11eb-9d37-01aa3dbc16c7.png)
> 

Try by removing the text `productivity meter` and enlarging the logo to the required dims

**Comment by Nishant2907** [comment]
okay, btw Happy new year.

**Comment by Nishant2907** [comment]
Tried what you said, it looks like this
![image](https://user-images.githubusercontent.com/59792198/103443487-c5555f80-4c85-11eb-87e9-ab109af0229a.png)

**Comment by Raghwendra-Dey** [comment]
@Nishant2907 try the other logo in the same way.

**Comment by Raghwendra-Dey** [comment]
> Tried what you said, it looks like this
> ![image](https://user-images.githubusercontent.com/59792198/103443487-c5555f80-4c85-11eb-87e9-ab109af0229a.png)

Hey @Nishant2907 are you done with the other logo in this format. Could you please post a photo of that here, so that we could decide upon?

**Comment by Nishant2907** [comment]
How about now?
![image](https://user-images.githubusercontent.com/59792198/103802887-d7644300-5075-11eb-9fec-a1a0fde6d457.png)
